R/France.R

```diff
@@ -0,0 +1,142 @@
+#' France Class containing country specific attributes and methods
+#'
+#' @description Country specific information for downloading, cleaning
+#' and processing covid-19 region data for France.
+#'
+#' Data is sourced from \url{https://health-infobase.France.ca}.
+#' @details Inherits from `DataClass`
+#' @examples
```

Comment:
Missing export call (leading to class not being found by tests).

Reply:
Fixed with new commit.

Reply:
:+1: